examples/custom/gears/gear.ts

```diff
@@ -160,7 +160,7 @@ export namespace Gear {
 
   export type Mixins = LineVector & FillVector
 
-  export type Visuals = XYGlyph.Visuals & {line: Line, fill: Fill}
+  export type Visuals = XYGlyph.Visuals & {line: visuals.LineVector, fill: visuals.FillVector}
```

Comment:
This is why I think `Fill` should remain equivalent to `FillVector` for now (which is not the case in this PR AFAICT but please correct me if I am mistaken)

Reply:
With the naming convention in this PR, `visuals.Line` matches `mixins.Line` and `visuals.LineVector` matches `mixins.LineVector`, etc. I also think that given the extent of changes (in this and other PRs) to this area of bokehjs, then having a hard break instead of an inevitable semantic/behavior break, will be beneficial for the user. For now I added a migration note. Let's go back to this when all PRs for 2.3 are applied, then we will have a clearer picture whether this is worthwhile or not.